lib/parallel_guest_migration_base.pm

```diff
@@ -697,6 +697,10 @@ sub initialize_test_result {
 
     $self->set_test_run_progress;
     my @_guest_migration_test = split(/,/, get_var('GUEST_MIGRATION_TEST', ''));
+    # Remove 'virsh_live_native_p2p_manual_postcopy' if present and record a soft failure
+    if (is_sle('=15-sp6')) {
+        @_guest_migration_test = $self->filter_migration_tests(@_guest_migration_test);
+    }
```

Comment:
Why do you need this block in base module ?

Reply:
pass test result, or job fails